src/slic3r/GUI/Plater.cpp

```diff
@@ -2278,7 +2278,8 @@ void Sidebar::update_presets(Preset::Type preset_type)
         Preset& printer_preset = wxGetApp().preset_bundle->printers.get_edited_preset();
 
         bool isBBL = printer_preset.is_bbl_vendor_preset(wxGetApp().preset_bundle);
-        wxGetApp().mainframe->show_calibration_button(!isBBL);
+        // Explicitly show calibration button for BBL printers
+        wxGetApp().mainframe->show_calibration_button(true);
```

Comment:
```suggestion
-        // Explicitly show calibration button for BBL printers
-        wxGetApp().mainframe->show_calibration_button(true);
+        // Show calibration Menu for BBL printers if Develop Mode is on.
+        bool show_calibration = !isBBL || wxGetApp().app_config->get("developer_mode") == "true";
+        wxGetApp().mainframe->show_calibration_button(show_calibration);
```

Here's an option if you'd like to gate this behind developer mode.
It'll default to off if a BBL printer, but will show it if Develop Mode is enabled.

Reply:
I cant find this in latest release?

Latest release only has this in lines 2313 and 2314:

        bool isBBL = printer_preset.is_bbl_vendor_preset(wxGetApp().preset_bundle);
        wxGetApp().mainframe->show_calibration_button(!isBBL);

Reply:
> Here's an option if you'd like to gate this behind developer mode. It'll default to off if a BBL printer, but will show it if Develop Mode is enabled.

@logikal  Thanks for your modification. Could you modify it as suggested here so that it is only displayed in developer mode?